projects/packages/action-bar/src/class-action-bar.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Put your classes in this `src` folder!
+ *
+ * @package automattic/jetpack-action-bar
+ */
+
+namespace Automattic\Jetpack;
+
+/**
+ * Action_Bar class.
+ */
+class Action_Bar {
+	/**
+	 * Enqueue scripts for rendering Action Bar client.
+	 */
+	public function enqueue_scripts() {
+		if ( is_admin() || ! is_single() ) {
+			return;
+		}
+
+		Assets::register_script(
+			'jetpack-action-bar',
+			'build/action-bar.js',
+			dirname( __DIR__ ) . DIRECTORY_SEPARATOR . 'src', // A full path to a file or a directory inside a plugin.
+			array(
+				'dependencies' => array( 'wp-i18n' ),
+				'in_footer'    => true,
+				'textdomain'   => 'jetpack-action-bar',
+			)
+		);
+		Assets::enqueue_script( 'jetpack-action-bar' );
```

Comment:
Since you're immediately enqueuing the script, I believe you should be able to do this:

```suggestion
				'textdomain'   => 'jetpack-action-bar',
				'enqueue'      => true,
			)
		);
```

Reply:
👍